Eat_and_Greet/src/screens/OnboardingScreen/index.jsx

```diff
@@ -0,0 +1,12 @@
+import { StyleSheet, Button, Text, View } from 'react-native';
+
+function OnboardingScreen() {
+    return (
+      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
```

Comment:
This seems to be the same styling as the container found in other files. So once possible move into a general stylesheet and import.